### PR TITLE
updated babel

### PR DESCRIPTION
**Pull request by FatemehOstad**
Hi! 👋
This PR updates the version of babel to fix https://github.com/advisories/GHSA-p6mc-m468-83gw.
Let me know if you need anything else!

### CODE REVIEW

**Comment by nicolo-ribaudo** [comment]
Hi! Since we use `^` to specify dependencies, those new versions are already covered by existing ranges.

**Comment by StevenStreib** [comment]
@nicolo-ribaudo yes, the minor version updates are covered by the ranges specified in the current `package.json`. With the latest changes @FatemehOstad pushed, those specified ranges are not changed, so this should be okay to merge now?

**Comment by StevenStreib** [comment]
@nicolo-ribaudo yes, the minor version updates are covered by the ranges specified in the current `package.json`. With the latest changes @FatemehOstad pushed, those specified ranges are not changed, so this should be okay to merge now?

**Comment by nicolo-ribaudo** [comment]
Yes it would be safe to merge, but it's completely unnecessary to merge and do a release just for this PR. Users of this package can already update their (deep) dependencies to avoid the vulnerability.

**Comment by Andarist** [comment]
@nicolo-ribaudo that is generally true but usually it's pretty annoying to update transitive deps when you have a lock file and your direct deps don't actually change. A lot of people don't know how to do this properly.

**Comment by nicolo-ribaudo** [comment]
Oh well, ok. I think we will have to wait for @developit to release these changes though.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@babel/preset-modules",
-  "version": "0.1.5",
+  "version": "0.1.6",
```
Comment (nicolo-ribaudo) [inline package.json:3]: Can you revert this? It will be done when releasing.

**Comment by nicolo-ribaudo** [comment]
I'm really confused. This PR doesn't update any dependency in package.json.

**Comment by StevenStreib** [comment]
I am one of your users 🙃 
When I install the most recently published version of `@babel/preset-modules` in my project, the versions of your dependencies that get installed (based on your lockfile) contain security vulnerabilities.
This PR updates your lockfile so that (after a release) when I update to the latest version of `@babel/preset-modules`, I will no longer be installing versions of your dependencies that have known security vulnerabilities.

**Comment by nicolo-ribaudo** [comment]
I have good (or bad, depending on how you see :stuck_out_tongue:) news for you!

Libraries don't publish lockfiles, and even if they accidentally do package manages will ignore those lockfiles and only consider your top-level one. This PR does not affect you in any way.

You can delete the various `@babel/*` dependencies that you see in your lockfile, re-run `yarn install`/`npm install`/`pnpm install`, and your package manager will automatically update them to the latest version. Some package managers support a command to update deep dependencies, for example `npm update --depth 999`, but "manually deleting them from the lockfile" always works regardless of what package manager you are using.

**Comment by StevenStreib** [comment]
OMG I feel so silly 🙈 
Thank you so much for explaining this, and apologies for my ignorance / lack of knowledge!

**Comment by developit** [comment]
lol this was a fun read 📦

**Pull request closed without merging**